get-bucket-sizes-for-org.sh

```diff
@@ -10,6 +10,7 @@ function usage {
     echo
     echo "  Options:"
     echo "      -e     :     Use existing credentials for AWS, rather than service binding credentials"
+    echo "      -p     :     Bucket prefix. Requires -e, and allows getting sizes for all buckets even if they're not bound"
```

Comment:
You should clarify what that means because I'm looking at it and I'm not sure what it means lol.

Reply:
I tried to clarify it a little.